GifSearcher/GifSearcher/SearchScene/SearchViewController.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  SearchViewController.swift
+//  GifSearcher
+//
+//  Created on 6/4/20.
+//
+
+import UIKit
+
+/**
+ Enum to display error messages on text field
+ */
+private enum SearchDisplayResults: String {
+    case noResults = "No Results Found"
+}
```

Comment:
I technique I've started using/liking for managing strings and their localization is to create `fileprivate extension String` blocks that define the strings needed for each file as `static var`'s on `String`:

```
fileprivate extension String {
    static var noResults: String { return "No Results Found" }
}
```
and then client code, instead of using the longer `SearchDisplayResults.noResults.rawValue`, can just use `.noResults` and all the strings for the file are consolidated w/in the `extension` on `String`. Similarly, more widely used strings can live in a non `fileprivate` `extension` in an appropriate file. In addition to shorter references in the client code, you get auto-completion of all strings available w/in scope.